airflow/config_templates/config.yml

```diff
@@ -2558,6 +2558,13 @@ triggerer:
       type: string
       example: ~
       default: "1000"
+    job_heartbeat_sec:
+      description: |
+        How often to heartbeat the Triggerer job to ensure it hasn't been killed.
+      version_added: 2.6.3
```

Comment:
Since we are adding a new configuration, this should be moved to 2.7.0

Reply:
Not necessarily. I think we do not have a rule "do not add new configuration options" in patchlevel.  We have the rule of not adding new features, but new configuration might be added in order to implement a bugfix (which I think is the case here). 

I think it's OK (and we've done that in the past) that we introduced new configuration in patchlevel version if they served bugfix purpose. There are a few configs already that were added in non- `.0` versions (smtp ones for example).

Reply:
(and yes, that was also my initial reaction, but I thought about it and I looked at the past configuration entries we created and it's not obvious that "new configuration == new feature".

Reply:
@ephraimbuddy Do you agree with @potiuk, or should I move it to 2.7.0?